sfs/array.py

```diff
@@ -135,6 +139,8 @@ def linear_random(N, min_spacing, max_spacing, center=[0, 0, 0],
         x0, n0, a0 = sfs.array.linear_random(12, 0.15, 0.4, orientation=[0, -1, 0])
         sfs.plot.loudspeaker_2d(x0, n0, a0)
         plt.axis('equal')
+        plt.xlabel('x / m')
+        plt.ylabel('y / m')        
```

Comment:
sfs/array.py:143:28: W291 trailing whitespace

Please enable https://editorconfig.org/ in your editor.